### DISCUSSION TITLE
Converting Throttle / Stopping Repeated Input?

### DISCUSSION THREAD

**Discussion by SarahSzabo** [AntiMicroX/antimicrox#708] [General]
Hello everyone,

I'm using Antimicrox for my Flight Sticks to do flashcards on a program called Anki and it works very well!
One problem I've been encountering is that certain buttons have a .1s refresh rate when held, and this causes issues for me since I'm clicking a D pad on the side to pause a video. Usually I can't get exactly less than .1s.

Is there a way to set the click in such that it only gives a single "space" character no matter how long it is held? Or at least to increase the time to like 1s or something? I checked the advanced menu, but the documentation appers to be too sparse for me to know how to solve this on my own.

Thank you for the support! :) ^^

**Comment by mrbeauregarde** [comment]
https://github.com/AntiMicroX/antimicrox/wiki/Advanced-Slot-Explanation#release
